packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.props.ts

```diff
@@ -14,6 +14,15 @@ export interface ScrollChange {
     end: boolean;
 }
 
+/**
+ * Horizontal overflow interface for consumers
+ * Includes all properties found in ScrollChange
+ * 'overflow' is true when there are enough items to cause overflow
+ */
+export interface HorizontalOverflowChange extends ScrollChange {
```

Comment:
It extends the ScrollChange so it includes the start and end and anything else provided by that interface

Reply:
I see, missed that.

Does "overflow" add value? I'm not sure it's worth adding a third property when the first two already communicate that information.

It's probably okay now - but I'm not sure this should extend scroll. scroll might have information like distance that would not be present in an overflow change event. You might consider extending the other direction.

Reply:
@chrisdholt for backup - we had several conversations about how the overflow callback would include scroll information as well. I thought we all agreed this data is important to the consumer on an overflow change.

Reply:
Yes. We had several conversations about how these handlers might (and do) have common properties. 

I'm just saying the two events are inherently different and that scrolling is likely (in the future) to contain information about a scroll event that cant be known in an overflow event because overflow events are not necessarily the product of scrolling , so the relationship seems backwards to me.

Reply:
Resolving for now...